contrib/intel/jenkins/tests.py

```diff
@@ -1062,7 +1062,7 @@ def __init__(self, jobname, buildno, testname, hw, core_prov, fabric,
                                             else 0
 
         if util_prov:
-            self.prov = f'{self.core_prov}\;{self.util_prov}'
+            self.prov = f'{self.core_prov};{self.util_prov}'
```

Comment:
Didn't we decide we need the '\' here? I thought it was making the command line think this was two different commands separated by the ';'?

Reply:
Yes, I'll fix that but I am more inclined towards using self.core_prov since we are only using verbs.

Reply:
I think its better to have it this way even if we are only testing verbs. That way if we add more things in the future we won't have to figure out the same bug again.